boskos/cluster/boskos-deployment.yaml

```diff
@@ -68,7 +68,7 @@ spec:
       terminationGracePeriodSeconds: 30
       containers:
       - name: boskos
-        image: gcr.io/k8s-prow/boskos/boskos:v20190730-dbac84bd7
+        image: gcr.io/k8s-staging-boskos/boskos:v20200819-984516e
```

Comment:
@chizhg I saw the knative PR you referenced is using /k8s-prow/ image, why do we switch to use k8s-staging-boskos?

Reply:
Boskos has been moved to a separate repo in https://github.com/kubernetes-sigs/boskos, and k8s-staging-boskos is the new registry it's using, plus Knative has also switched to it - https://github.com/GoogleCloudPlatform/oss-test-infra/blob/master/prow/knative/cluster/build/400-boskos-deployment.yaml#L36

Reply:
I see, thanks for your explanation!